.orchestra/support/revng-distributable/install.sh

```diff
@@ -8,6 +8,7 @@ mkdir -p "${DESTDIR}${ORCHESTRA_ROOT}/revng/root"
 cd "$ORCHESTRA_ROOT"
 
 for component in "$@"; do
```

Comment:
Usually we use capital names for variables in bash, for next time. No need to fix this now.

Reply:
Noted